<a href="https://colab.research.google.com/github/kennygiovanni/Checkpoint-AVD/blob/main/CheckPoint%201%20AVD.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#**Data Penjualan Produk Cetakan**

#Import Library


In [1]:
import pandas as pd

#Memuat Dataset

In [5]:
df = pd.read_csv("/data_penjualan.csv")
df

Tanggal;Jenis Produk;Jumlah Order;Harga;Total
0           05/08/2022;Foodpak260;1000;1800;1800000
1      05/08/2022;FoodpakMatte245;1000;1900;1900000
2      05/08/2022;CraftLaminasi290;5000;750;3750000
3     05/08/2022;CraftLaminasi290;1000;1200;1200000
4           07/08/2022;Dupleks310;1000;1550;1550000
...                                             ...
1071            14/11/2023;Ivory230;1000;900;900000
1072   15/11/2023;CraftLaminasi290;2000;800;1600000
1073   15/11/2023;CraftLaminasi290;1500;875;1312500
1074      15/11/2023;FoodpakMatte;1000;2200;2200000
1075         15/11/2023;GreaseProof;1000;300;300000

[1076 rows x 1 columns]

Tanggal
Jumlah baris: 1076
Tipe data: object
Deskripsi: Tanggal transaksi penjualan cetakan

Jenis Produk
Jumlah baris: 1076
Tipe data: object
Deskripsi: Jenis produk cetakan yang terjual

Jumlah Produk
Jumlah baris: 1076
Tipe data: int64
Deskripsi: Jumlah unit produk yang terjual pada tanggal tertentu

Harga
Jumlah baris: 1076
Tipe data: int64
Deskripsi: Harga dari produk tertentu

Total
Jumlah baris: 1076
Tipe data: int64
Deskripsi: Total nilai penjualan dalam mata uang lokal (misalnya, Rupiah) untuk setiap transaksi



# Business Understanding

##Business Objective

Dataset ini mencakup rincian transaksi penjualan seperti waktu, jenis produk, volume pesanan, harga satuan, dan total nilai penjualan dalam periode tertentu. Meskipun data tersedia, perusahaan belum memiliki informasi akurat mengenai produk dengan performa terbaik. Akibatnya, penentuan produk unggulan untuk strategi promosi dan pengelolaan stok masih bersifat estimasi, sehingga pengambilan keputusan menjadi kurang optimal dalam meningkatkan laba.

Analisis ini bertujuan untuk mengidentifikasi produk dengan performa penjualan tertinggi berdasarkan jumlah pesanan dan total pendapatan. Hasil analisis ini akan menjadi landasan bagi perusahaan dalam mengoptimalkan strategi pemasaran serta manajemen stok pada produk-produk prioritas.


## Assess Situation

Upaya perusahaan percetakan buat meningkatkan keuntungan terhambat karena mereka belum mempunyai info akurat soal produk mana yang performanya paling bagus. Padahal, penentuan produk unggulan itu sangat penting. Lewat analisis data penjualan ini, kita harap bisa mendapatkan gambaran produk mana yang paling potensial untuk jadi acuan dalam menyusun strategi bisnis ke depannya.

## Analytic Goals

Untuk mendapatkan jenis produk unggulan agar perusahaan dapat memfokuskan strategi promosi dan penyediaan pada produk unggulan, perlu dilakukan:

1. Mengidentifikasi produk dengan jumlah order tertinggi.

2. Menghitung total pendapatan dari setiap jenis produk.

3. Mengelompokkan produk berdasarkan tingkat performa penjualan menjadi produk unggulan dan non-unggulan.

## Project Plan

1. Business Understanding
Menentukan tujuan analisis untuk mengetahui produk dengan performa penjualan terbaik sebagai dasar strategi promosi dan pengelolaan stok.

2. Data Understanding
Memahami struktur dan karakteristik data penjualan serta mengecek kelengkapan dan konsistensi data.

3. Data Preparation
Membersihkan dan menyiapkan data agar siap dianalisis, termasuk perbaikan format dan perhitungan total pendapatan per produk.

4. Visualisasi & Dashboard
Membuat grafik atau dashboard untuk menampilkan perbandingan jumlah order dan pendapatan tiap produk.

5. Analisis & Insight
Mengidentifikasi produk unggulan dan non-unggulan serta menyusun insight untuk mendukung pengambilan keputusan.